src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,40 @@
-import React from 'react';
+import { FC } from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: FC<Props> = ({
+  comment: {
+    email,
+    name,
+    body,
+  },
+}) => {
+  const emailString = `mailto:${email}`;
```

Comment:
You can memorize this variable to avoid redefining it on every rerender but only when email has been changed 
```suggestion
  const emailString = useMemo(() => `mailto:${email}`, [email]);
```